### DISCUSSION TITLE
There are `KerasModelArtifact` and `TensorflowSavedModelArtifact` here, which one should I use?

### DISCUSSION THREAD

**Discussion by bojiang** [bentoml/BentoML#1186] [Q&A] [answered]
As title.

**Comment by parano** [comment, marked as answer]
For others who have the same question - this is being addressed as part of our effort around better Tensorflow integration https://github.com/bentoml/BentoML/issues/1188

Going forward we will make it more clear which model artifact to use depending on the Tensorflow API used for creating the model.

As a rule of thumb for now(in BentoML <= 0.9.2), if you are using Tensorflow Keras API, go with `KerasModelArtifact`, otherwise, use the `TensorflowSavedModelArtifact`.
